R/sharpen.R

```diff
@@ -1,13 +1,86 @@
-#' Return a sharpened image
```

Comment:
Maybe we could add the author and a title at the top.

Reply:
No problem.